Formula/orientdb.rb

```diff
@@ -48,6 +43,16 @@ def post_install
     (var/"log/orientdb").mkpath
     touch "#{var}/log/orientdb/orientdb.err"
     touch "#{var}/log/orientdb/orientdb.log"
+
+    # set the server's root user  password to changeme
+    ENV["ORIENTDB_ROOT_PASSWORD"] = "changeme"
```

Comment:
What's the default value? We install e.g. `mysql` with no password by default so it'd be nice to emulate that.